docs/source/postprocessing/openPMD.rst

```diff
@@ -0,0 +1,6 @@
+.. _pp-openPMD:
+
+openPMD
+=======
+
+Please see https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Post-processing-and-Visualization for now.
```

Comment:
I can also paste that list of links directly, but I did not start migrating the wiki yet